_dashboards/visualize/geojson-regionmaps.md

```diff
@@ -1,19 +1,19 @@
 ---
 layout: default
-title: Region map visualizations
-nav_order: 7
+title: Using Region Map
+parent: Visualization types
```

Comment:
Do we have page called Visualization Types created yet?

Reply:
@Naarcha-AWS Yes, see PR #2216